lib/result/object/TingClientObject.php

```diff
@@ -1,7 +1,36 @@
 <?php
 
 class TingClientObject {
-	public $id;	
+	public $id;
 	public $data;
+	public $relationsData;
+
+  /**
+   * We do not want specific items in the review box on bibliotek.dk.
+   * Therefore we need to be able to remove them. Such as 150005-anmeld reviews.
+   *
+   * @param array $needle
+   * $needle must be an assoc array. ['field_name' => 'field_value'].
+   * Ex: array('relationUri' => '150005-anmeld')
+   *
+   * @return array
+   * The trimmed relationsData array.
+   */
+	public function getTrimmedRelationsData($needle) {
```

Comment:
Skal getTrimmedRelationsData ikke kaldes et sted, eller der der noget "magi" der sørger for det?

Reply:
Jeg fik ikke tilføjet jer til den anden MR: https://github.com/DBCDK/bibdk/pull/138

Det er I nu.